archaius2-api/src/main/java/com/netflix/archaius/api/exceptions/ConfigException.java

```diff
@@ -15,7 +15,7 @@
  */
 package com.netflix.archaius.api.exceptions;
 
-public class ConfigException extends Exception {
+public class ConfigException extends RuntimeException {
```

Comment:
ConfigException isn't very specific, essentially 'GeneralArchaiusFault'.  How about subtypes for ValueConversionException, DuplicateKeyException, ConfigLoaderException, etc.